website/docs/controller_101/README.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
Suggest this goes in `concepts/` with other service overviews

Reply:
Done here: df2b274e